tests/bento_service_examples/paddle_service.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
```

Comment:
Files in this directory are only for unit tests.

Reply:
Ah I see - will remove.